A72-open-telemetry-tracing.md

```diff
@@ -205,57 +186,75 @@ On attempt span:
   boolean value indicating whether the stream is undergoing a transparent retry.
 * If the RPC experienced load balancer pick delay, add an Event with the name 
   "Delayed LB pick complete" upon creation of the stream on the transport. 
-* When the application sends an outbound message to the transport, add an Event 
+* When the application sends an outbound message to the transport, add Event(s)
+  (it depends on implementation whether there is a single event or two separate 
+  events for compressed/uncompressed message sizes, the same below) 
   with name "Outbound message sent" and the following attributes:
   * key `message.event.type` with string value "SENT".
-  * key `message.message.id` with integer value of the seq no. The seq no. is a sequence
-    of integer numbers starting from 0 to identify sent messages within the stream, the same below.
+  * key `message.message.id` with integer value of the seq no. The seq no. indicates
+    the order of the sent messages on the attempt (i.e., it starts at 0 and is 
+    incremented by 1 for each message sent), the same below.
   * key `message.event.size.uncompressed` with integer value of uncompressed 
     message size. The size is the total attempt message bytes without encryption,
     not including grpc or transport framing bytes, the same below.
-  * If any compression, key `message.event.size.compressed` with integer value
-    of compressed message size.
-* When an inbound message has been received from the transport, add an Event
-  with name "Inbound message read" and the following attributes:
-  * key `message.event.type` with String value "RECEIVED".
-  * key `message.message.id` with integer value of the seq no.
-  * key `message.event.size.compressed` with integer value of wire message size. 
-  * If any compression, key `message.event.size.uncompressed` with integer value
-    of uncompressed message size.
+  * If compression needed, add key `message.event.size.compressed` with integer 
+    value of compressed message size.
+* When an inbound message has been received from wire, add Event(s) with name
```

Comment:
If we're intentionally being imprecise about where in the stack we record this event, then let's not say "from wire".

Reply:
+1